src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java

```diff
@@ -507,7 +507,31 @@ public enum Skins {
     // https://minecraft-heads.com/custom-heads/food%20&%20drinks/24965-pepperoni
     PEPPERONI("10edb47f91bdb5af0fdf3290c7d181163f5df72a3b1c7beda1d0692e36c158da"),
     // https://minecraft-heads.com/custom-heads/food%20&%20drinks/60305-pizza-dough
-    DOUGH("8166d79de64abb4457ab87fa8f6706d55ce682e10db849ea6dd124095b12276e");
+    DOUGH("8166d79de64abb4457ab87fa8f6706d55ce682e10db849ea6dd124095b12276e"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/398-ramen-red-bowl
+    RAMEN_BEEF("36411870d9832c88188669c86dd8c68068b8466b38e98cedcf8eeb75cfb3"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/399-ramen-green-bowl
+    RAMEN_CHICKEN("726a14f71eff6a89551a74db3811a84d17c59657c9d5176431a23a35f951af"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/400-ramen-blue-bowl
+    RAMEN_SEAFOOD("e153afca9abca55ec359cfaa5aacb0101aaafb75b7da63b63b6a1ea5a58873a"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/59641-ramen-black-bowl
+    RAMEN_HEARTY("9b54225d7a2078319d886a7751c9147c6b7e13bb0aa0ca18cdce01357bc65b3a"),
+    // https://minecraft-heads.com/custom-heads/plants/8691-coconut-brown
+    GREEN_ONION("c2dd5433db4fddebc4a77166735699400cb18d43672ab31326a83f0b7c2586cc"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/54977-meatloaf
+    MEATLOAF("98eeafdf6d7c6c9943e44bfcfdbe037d6c15d6d5e354c3ccf72bf886cde4e1c6"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/59814-johnsonville-italian-sausage
+    SAUSAGE("49d6afe4e80fd6a70e4833852062f6d5a2abf0da007fbf38b31fe4048b3cf9bd"),
+    // https://minecraft-heads.com/custom-heads/plants/6358-potato
+    TATER_TOT("dbfb6d4a54c17e2748437acc7098fbb1a3a12a407f51b3e49542332714846fd8"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8748-bagel
+    BAGEL("502e92f13de3bee69228c384478e761230681e5fce9bda195daeaf8484139331"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8750-poppyseed-bagel
+    BAGEL_EVERYTHING("653aaae67dda225b78e82886d046c67fdf37f1bbe2dfc55f1324781f8159f5"),
```

Comment:
Bagel everything isn’t that descriptive

Reply:
Everything bagel is, annoyingly, the name of a type of bagel, and marking up with the archetype first followed by the descriptor is in line with my code style, so while it seems to not make sense it annoyingly does. everything Bagel, 'murica